### PR TITLE
simplify the body check flow

### PR DESCRIPTION
**Pull request by gasolin**
simplify body check flow since we already checked `isOnGitHub` at the top. Also passed `url` for process from setInterval function.

All function should work the same as 0.904 version

### CODE REVIEW

**Comment by gasolin** [comment]
@owocki @jclancy93 could you help review this?

**Comment by jclancy93** [comment]
This all looks good to me, thanks for doing this. The body function definitely needed to be cleaned up.

File: src/script/pageload/body.js [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
-            injectGetBountyAmount();
-    } else if (isOnRepo){
+    if (isOnIssuePage) {
+        addButtonToIssuePage();
+        addBountyInfoToIssuePage(url);
+        injectGetBountyAmount();
+    } else if (isOnRepo) {
         var repoUrl = document.location.href.split('issues')[0].split('pulls')[0];
         injectGetNumberBounties(repoUrl);
-    } else if (isOnGitHub){
```
Comment (jclancy93) [inline src/script/pageload/body.js]: good catch

**Comment by gasolin** [comment]
@owocki since @jclancy93 already reviewed this, could you help merge it?

**Comment by owocki** [comment]
thanks @gasolin !

**Pull request merged**